jax/_src/scipy/stats/bernoulli.py

```diff
@@ -36,3 +36,26 @@ def logpmf(k: ArrayLike, p: ArrayLike, loc: ArrayLike = 0) -> Array:
 @_wraps(osp_stats.bernoulli.pmf, update_doc=False)
 def pmf(k: ArrayLike, p: ArrayLike, loc: ArrayLike = 0) -> Array:
   return jnp.exp(logpmf(k, p, loc))
+
+@_wraps(osp_stats.bernoulli.cdf, update_doc=False)
+def cdf(k: ArrayLike, p: ArrayLike) -> Array:
+  k, p = jnp._promote_args_inexact('bernoulli.cdf', k, p)
+  zero, one, one_minus_p = _lax_const(k, 0), _lax_const(k, 1), _lax_const(k, 1-p)
+  conds = [
+    jnp.isnan(k) | jnp.isnan(p) | (p < zero) | (p > one),
+    lax.lt(k, zero),
+    jnp.logical_and(lax.ge(k, zero), lax.lt(k, one)),
+    lax.ge(k, one)
+    ]
+  vals = [jnp.nan, zero, one_minus_p, one]
+  return jnp.select(conds, vals)
+
+@_wraps(osp_stats.bernoulli.ppf, update_doc=False)
+def ppf(q: ArrayLike, p: ArrayLike) -> Array:
+  q, p = jnp._promote_args_inexact('bernoulli.ppf', q, p)
+  zero, one, one_minus_p = _lax_const(q, 0), _lax_const(q, 1), _lax_const(q, 1-p)
```

Comment:
The tracer conversion error is coming from `_lax_const(q, 1-p)`. Since `p` is a traced value, a constant cannot be derived from it.